### ISSUE TITLE
Add description to search box

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5792]
Hotlist: https://buganizer.corp.google.com/hotlists/5891206
Bug: https://buganizer.corp.google.com/issues/338006277

### Observed Results

The "search" box does not have a clear description which explains its purpose.

* [Screencast](https://screencast.googleplex.com/cast/NDUzMjE2MDc0MzYwNDIyNHw1NmYzZGNiMC0wNQ).

### Expected Results

The search box should have a clear description, for example "search for articles in google".

### User Impact

Users will get confused about what information they need to fill in the search field to find their required information. this will affect users who depend on screen readers, users such as who are low vision, blind.

### Steps To Reproduce

1. Enable ChromeVox using <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>z</kbd>.
1. Go to https://dart.dev/search?cx=011220921317074318178%3A_yy-tmb5t_i&ie=UTF-8&hl=en&q=isolate.
1. Using the tab key navigate to "search" field.
1. Observe screen reader announcement.

### Relevent GAR Sheet

[A11Y Tracker](https://a11ytracker.corp.google.com/products/034ba691-ac78-4cc2-88c2-e06449cfd17e/edit/testresults?useCase=7jg80pq64n0p8&e=ProductLabels).

### GAR Details

[1.11. Is the purpose of each link, control, and button clear?](http://go/gar-web-purpose-link-control-button).

### Environment Details

* Machine: Chromebook.
* Browser: Chrome (123.0.6312.112 (Official Build) (64-bit)).
* Accessibility Tool: ChromeVox.
* Bug Filling Time: 10 mins.

**Comment by yashathwani** [comment]
Would like to work on this issue can you assign this to me

**Comment by yashathwani** [comment]
@atsansone Any idea of what should be displayed in the search field!!

**Comment by parlough** [comment]
These search results are generated, styled, and managed by the Google Custom Search setup we use.

Since we don't control that, I'm going to close this in favor of an umbrella issue https://github.com/dart-lang/site-www/issues/6486 which tracks implementing a new experience that we can properly integrate and style. Thanks!
